Assets/StreamingAssets/Data/Prototypes/SettingsTemplate.json

```diff
@@ -187,6 +187,17 @@
               "MaximumValue": "2"
             },
             "Tooltip": "ui_scale_tooltip"
+          },
+          {
+            "Name": "ui_animation_speed",
+            "Key": "video_general_ui_animation_speed",
+            "DefaultValue": "5",
+            "ClassName": "UIAnimationSpeedSlider",
```

Comment:
Since you don't need any custom logic just change it to `GenericSlider`

Reply:
You right, I was thinking about some logic(Apply speed function in controller or something like that) but I will leave it for now. Thank you for reminding that.